{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--74d6b128-9186-4c44-93d0-7b3caa9b6c70",
    "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
    "created": "2022-04-01T23:15:00.000Z",
    "modified": "2022-04-13T13:58:00.000Z",
    "name": "CVE-2021-3461",
    "description": "A flaw was found in keycloak where keycloak may fail to logout user session if the logout request comes from external SAML identity provider and Principal Type is set to Attribute [Name].",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2021-3461"
        },
        {
            "source_name": "cwe",
            "external_id": "CWE-613"
        },
        {
            "source_name": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565",
            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565"
        },
        {
            "source_name": "cve2stix",
            "description": "This object was created using cve2stix from the Signals Corps.",
            "url": "https://github.com/signalscorps/cve2stix"
        }
    ],
    "object_marking_refs": [
        "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
    ],
    "extensions": {
        "extension-definition--b2b5f2cd-49e6-4091-a0e0-c0bb71543e23": {
            "extension_type": "property-extension",
            "impact": {
                "baseMetricV2": {
                    "acInsufInfo": false,
                    "cvssV2": {
                        "accessComplexity": "MEDIUM",
                        "accessVector": "LOCAL",
                        "authentication": "NONE",
                        "availabilityImpact": "NONE",
                        "baseScore": 3.3,
                        "confidentialityImpact": "PARTIAL",
                        "integrityImpact": "PARTIAL",
                        "vectorString": "AV:L/AC:M/Au:N/C:P/I:P/A:N",
                        "version": "2.0"
                    },
                    "exploitabilityScore": 3.4,
                    "impactScore": 4.9,
                    "obtainAllPrivilege": false,
                    "obtainOtherPrivilege": false,
                    "obtainUserPrivilege": false,
                    "severity": "LOW",
                    "userInteractionRequired": true
                },
                "baseMetricV3": {
                    "cvssV3": {
                        "attackComplexity": "LOW",
                        "attackVector": "LOCAL",
                        "availabilityImpact": "NONE",
                        "baseScore": 7.1,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "REQUIRED",
                        "vectorString": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:N",
                        "version": "3.1"
                    },
                    "exploitabilityScore": 1.8,
                    "impactScore": 5.2
                }
            },
            "lastModifiedDate": "2022-04-13T13:58Z",
            "nvd_cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2021-3461"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "A flaw was found in keycloak where keycloak may fail to logout user session if the logout request comes from external SAML identity provider and Principal Type is set to Attribute [Name]."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "en",
                                    "value": "CWE-613"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1941565",
                            "refsource": "MISC",
                            "tags": [
                                "Issue Tracking",
                                "Patch",
                                "Third Party Advisory"
                            ]
                        }
                    ]
                }
            },
            "publishedDate": "2022-04-01T23:15Z"
        }
    }
}